Rhinoceros_Engine/Convert/ToBHoM.cs

```diff
@@ -357,10 +364,21 @@ public static BHG.NurbsSurface ToBHoM(this RHG.NurbsSurface surface)
 
         /***************************************************/
 
-        public static BHG.PolySurface ToBHoM(this RHG.Brep brep)
+        public static BHG.ISurface ToBHoM(this RHG.Brep brep)
         {
             if (brep == null) return null;
 
+            if (brep.Surfaces.Count == 0) return null;
+
+            // PlanarSurface case
+            if (brep.IsPlanarSurface())
+            {
+                BHG.ICurve externalEdge = RHG.Curve.JoinCurves(brep.DuplicateNakedEdgeCurves(true, false)).FirstOrDefault().ToBHoM();
```

Comment:
Wouldnt it be quicker to only join the curves one time, rather than here AND on the row below?

Also, shouldn't the first curve be skipped for the internal edges?

Reply:
The former line returns the outer edge, whereas the second one returns the inner ones. 
I am joining the former because it returns them exploded (list of lines in case of polylines)

Reply:
Ah! Sorry, I missed the difference in the boolean inputs! Both rows looked the same at first glance!

Reply:
No worries! I am sure there is a more performing way to do it, but I haven't found it quickly enough and resorted to this.